Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -169,7 +166,7 @@ public class KolodaView: UIView, DraggableCardDelegate {
                 
                 for index in 0..<countOfNeededCards {
                     let nextCardContentView = dataSource.koloda(self, viewForCardAtIndex: UInt(index+currentCardNumber))
-                    let nextCardView = DraggableCardView(frame: frameForCardAtIndex(UInt(index)))
+                    let nextCardView = DraggableCardView(frame: frameForCardAtIndex(0))
```

Comment:
@AEugene how do you feel about creation an additinal method frameForTopCard() ? I guess it will indicate your intention much clearly.